components/colorhandle/stories/template.js

```diff
@@ -12,18 +11,19 @@ export const Template = ({
 	colorHandleStyle = {
 		"--spectrum-picked-color": "rgba(255, 0, 0, 0.5)",
 	},
-	// ...globals
+	...globals
 }) => {
+
+	const checkerboardContent = html `<div class="${rootClass}-inner"></div>`
+
 	return html`
-  <div class=${classMap({
-		[rootClass]: true,
-		"is-disabled": isDisabled,
-		"is-focused": !isDisabled && isFocused,
-		...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
```

Comment:
I think we lost these classes being applied, now that this story is using OpacityCheckerboard. These also need to be included with `customClasses`?